build/Linux+BSD/portable/copy-libs

```diff
@@ -54,9 +56,9 @@ getCrossPlatformDependencies() {
   # Qt base dependencies (see "mscore/CMakeLists.txt")
   dest_dir="$qt_dest/lib"
   #dest_dir="$lib_dest"
-  copyLib libicudata.so.42 #icudt53.dll # Needed by Ubuntu 12.04
-  copyLib libicui18n.so.42 #libicuin53.so.5 # Needed by Ubuntu 12.04
-  copyLib libicuuc.so.42 #libicuuc53.so.5 # Needed by Ubuntu 12.04
+  copyLib libicudata.so.52 #icudt53.dll # Needed by Ubuntu 12.04
+  copyLib libicui18n.so.52 #libicuin53.so.5 # Needed by Ubuntu 12.04
+  copyLib libicuuc.so.52 #libicuuc53.so.5 # Needed by Ubuntu 12.04
```

Comment:
Try removing them so it's just ".so" and see if the AppImages still build & run ok.